content/zh/blog/2023/dlb-connection-balancing/index.md

```diff
@@ -0,0 +1,325 @@
+---
+title: "在 Istio 中使用加速卸载连接负载均衡"
+description: "在 Istio 网关中使用 DLB 连接负载均衡配置来加速连接均衡。"
+publishdate: 2023-06-08
```

Comment:
BTW just saw this https://github.com/istio/istio.io/pull/13707

Reply:
OK, wait for that merged and then update this PR.